src/main/webapp/i18n/de/lectureUnit.json

```
@@ -114,6 +114,33 @@
             "created": "Eine neue Dateieinheit wurde erstellt",
             "updated": "Dateieinheit aktualisiert",
             "notReleasedTooltip": "Nur sichtbar für Tutor:innen und Lehrende. Veröffentlichungsdatum:",
+            "createAttachmentUnits": {
+                "noUnitDetected": "Es wurde keine Einheit erkannt und konnte nicht automatisch geteilt werden. Bitte fügen Sie Informationen für jede Einheit hinzu!",
+                "pageTitle": "Prozesseinheiten",
+                "addUnit": "Einheit hinzufügen",
+                "processUnits": "Prozesseinheiten",
+                "processAutomatically": "Automatische Einheitenverarbeitung",
+                "validation": {
+                    "empty": {
+                        "unitName": "Unit name can not be empty",
+                        "startPage": "Die Startseite darf nicht leer sein",
+                        "endPage": "Endseite darf nicht leer sein"
+                    },
+                    "endPageLower": "Endseite darf nicht kleiner als 1 sein",
+                    "startPageBigger": "Die Startseite darf nicht größer als",
+                    "startPage": "Startseite muss größer oder gleich 1 sein",
+                    "endPage": "Endseite darf nicht größer als"
```

```suggestion
                    "endPage": "Letzte Seite darf nicht größer als {{max}} sein."
```
I expect some number is added to the end of this error message? In that case a ‘sein’ after it would be missing.
I think you can use placeholders like in the suggestion to set variable data within the translation. There should be examples in the Artemis code.

The same then applies to `startPageBigger` as well.